### PR TITLE
chore: add new notes about PRs in babel

### PR DESCRIPTION
**Pull request by armano2**

### CODE REVIEW

File: tests/ast-alignment/fixtures-to-test.ts [hunk 1]
```diff
@@ -383,12 +383,16 @@ tester.addFixturePatternConfig('typescript/basics', {
     'directive-in-module',
     'directive-in-namespace',
     /**
-     * Babel range bug, which will be fixed by https://github.com/babel/babel/pull/9284
+     * PR for BigInt support has been merged into Babel: https://github.com/babel/babel/pull/9284
```
Comment (JamesHenry) [inline tests/ast-alignment/fixtures-to-test.ts]: This isn't related to BigInt is it?

Comment (armano2) [reply tests/ast-alignment/fixtures-to-test.ts]: nope typo xd

**Comment by armano2** [comment]
That's a little not related but TS7010 will be reported when `noImplictAny` option is set.
```ts
    /**
     * TypeScript does not report any diagnostics for this file, but Babel throws:
     * [SyntaxError: Unexpected token, expected "{" (2:8)
      1 | class Foo {
    > 2 |   foo?();
        |         ^
      3 |   bar?(): string;
      4 |   private baz?(): string;
      5 | }]
     */
    'class-with-optional-methods', // babel parse errors
```

**Comment by JamesHenry** [comment]
@armano2 I don't think we'll ever want to throw on type errors from this parser

**Comment by armano2** [comment]
ok, it's just information why :>

**Pull request merged**